src/test_l2_tvSettings.c

```diff
@@ -3307,7 +3307,8 @@ void test_l2_tvSettings_GetTVGammaTarget(void)
     for( colorTemp=tvColorTemp_STANDARD; colorTemp < tvColorTemp_MAX; colorTemp++ )
     {
         UT_LOG_DEBUG("Invoking GetTVGammaTarget() with colorTemp = %d", colorTemp);
-        GetTVGammaTarget(colorTemp, &x, &y);
+        status = GetTVGammaTarget(colorTemp, &x, &y);
+        UT_ASSERT_EQUAL_FATAL(status, tvERROR_NONE);
```

Comment:
Dont use UT_ASSERT_EQUAL_FATAL instead use UT_ASSERT_EQUAL

Reply:
addressed in commit id:- 45ba5edef84a528bb3ef3e891c40bc01d1646c0a